public/js/components/card-list.jsx

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var React = require('react');
+var CardItem = require('components/card-item');
+
+module.exports = React.createClass({
+  displayName: 'CardChoice',
```

Comment:
I think it's a little confusing that this is `CardChoice` but the file is `card-list.jsx` - maybe rename one or the other?